projects/qpid-proton/c_tests_fuzz_CMakeLists.patch

```diff
@@ -25,7 +25,7 @@ index b4470d59..211252c4 100644
 +    set(FUZZING_QPID_PROTON_CORE_LIBRARY qpid-proton-core)
 +  endif()
 +  target_link_libraries (${test} ${FUZZING_QPID_PROTON_CORE_LIBRARY} ${FUZZING_LIBRARY})
-+  # $LIB_FUZZING_ENGINE is a C++ library, which needs c++ std lib
++  # -lFuzzingEngine is a C++ library, which needs c++ std lib
```

Comment:
Does -lFuzzingEngine mean anything now ?

Reply:
Yeah it still works.

Reply:
This project still uses it by the way: https://github.com/apache/qpid-proton/blob/3b1edb510290d771c3c554ab634031c840a45691/c/tests/fuzz/CMakeLists.txt#L28